### ISSUE TITLE
[BUG] Capture Choice → Error when capturing to a Specific Note vs. Active Note

### ISSUE THREAD

**Issue by FelipeRearden** [chhoumann/quickadd#431]
Hello @chhoumann !!!!!

**Describe the bug**

I am facing a different behavior for two **Capture Choice ** that has the same output BUT the difference between these two Captures is that one of then is: 

🅰️ **Capture to the Active Note** 
🅱️ **Capture to a Specific File**
 
✅ Working perfect on 🅰️

❌ When I run 🅱️ I got this ERROR MESSAGE:
![E6528A21-F172-4667-A3B8-22A1DF93BBC8](https://user-images.githubusercontent.com/85180696/227539782-2185a045-cb46-4e97-8497-06e21dfed526.jpeg)

This is the Capture code → I am using a **inline script**

```js quickadd
const aTexts =   [ "Quotes", "Aphorisms", "info"];
const callouts = [ "### Quotes", "### Aphorisms", "info"];
const txt =   [ "Lorem", "Ipsum"];
const out = [ "### Lorem", "### Ipsum"];

let choice = await this.quickAddApi.suggester(aTexts, callouts);

if (choice === "info" ) {
let item = await this.quickAddApi.suggester(txt, out);
return item;
} else {
return choice;
}
```

**To Reproduce**

1. Create Capture Choice 🅰️
2.  Create Capture Choice 🅱️
3. Run 🅰️
4. Run 🅱️
5. See error after running 🅱️

**Expected behavior**
Choice 🅱️ should run exactly like Choice 🅰️

**Screenshots**
📸 🅰️ 
![FA285D42-FB46-4AC7-A84F-B68303D5BC6D](https://user-images.githubusercontent.com/85180696/227541607-868a156a-5e1c-4ecb-9655-a372049c148d.jpeg)

📸 🅱️
![A3DA9BB1-9D74-486A-9603-549ECFC93A91](https://user-images.githubusercontent.com/85180696/227541526-4892b361-f7ab-40a1-b0b9-5e5e783488e2.jpeg)

**Smartphone (please complete the following information):**
 - Device: IPad Pro 2nd gen 
 - OS: iPadOS 16.2
 - Browser Safari
 - Version I dont know :(

**Additional context**
Thank you very much for reading this 🙏

I wish you a fantastic day ☀️

**Comment by chhoumann** [comment]
Hey @FelipeRearden! Thank you for reporting this.

I've just attempted to recreate the issue, but cannot seem to.
Any chance this was fixed 'accidentally' in v0.18? :)

Or is there anywhere a variable is declared twice? I.e. `let item` appears twice? Or any attempt at assigning a `const` variable after it has been declared initially?
Just want to rule it out - even though it doesn't appear in the code you've provided.

Wishing you a great day as well!

**Comment by FelipeRearden** [comment]
> Hey @FelipeRearden! Thank you for reporting this.

Hello @chhoumann !!!! Thnank you very much for the reply :)

> I've just attempted to recreate the issue, but cannot seem to. Any chance this was fixed 'accidentally' in v0.18? :)

I think the Obsidian gods are in our side today. It's working prefect on version 0.18.1 🙏🙏🙏

Thanks for the reminder :)

> Wishing you a great day as well!

Me too. I hope you have a fantastic monday :)
